docs/release_notes/v1.11.5.md

```diff
@@ -1,6 +1,36 @@
 # Dapr 1.11.5
 
-This patch release includes a fix in gRPC-to-gRPC service invocation when the `ServiceInvocationStreaming` feature is enabled. Users who rely on gRPC-to-gRPC service invocation (legacy, non proxy-based) are encouraged to upgrade to Dapr 1.11.5 before upgrading existing deployments to 1.12.x, to prevent temporary issues during a rollout of Dapr 1.12.x.
+This update contains the following security fixes:
```

Comment:
Moved from below:

```suggestion
**Important:** Users who rely on gRPC-to-gRPC service invocation (legacy, non proxy-based) are encouraged to upgrade to Dapr 1.11.5 before upgrading existing deployments to 1.12.x, to prevent temporary issues during a rollout of Dapr 1.12.x.

This update contains the following security fixes:
```